test/karma.conf.js

```diff
@@ -50,27 +50,22 @@ module.exports = function(config) {
   config.set({
     basePath: '',
 
-    frameworks: ['browserify', 'qunit'],
+    frameworks: ['qunit'],
 
     autoWatch: false,
 
     singleRun: true,
 
     files: [
+      // include and execute the standalone test bundle first
+      '../build/temp/tests.js',
+
+      // then include video.js through globals to run the API tests
       '../build/temp/video-js.min.css',
       '../build/temp/video.min.js',
-      '../build/temp/tests.js',
+      'api/api.js'
     ],
 
-    preprocessors: {
-      '../test/unit/**/*.js': [ 'browserify' ]
-    },
-
-    browserify: {
-      debug: true,
-      transform: [ 'babelify', 'browserify-istanbul' ]
-    },
-
     plugins: [
```

Comment:
fwiw, we don't actually need the plugins field. Afaik, latest karma knows to look for plugins in npm and autoloads them. Something to think about/try out.